src/main/java/use_case/logout/LogoutInputData.java

```diff
@@ -0,0 +1,15 @@
+package use_case.logout;
+
+public class LogoutInputData {
+
+    private final String email;
+
+    public LogoutInputData(String email) {
```

Comment:
I thought we would use userID to determine which user to logout. Is there a reason we are using the user email? (just wondering this)

Reply:
tbh i think its not necessary to include both email and userID to log someone out. Since we keep track of the current user in our user dao, we could just set it to null.

We can also use the currentUser attribute in the use dao to do things like creating a post (the current user created the post) or determine if the current user made the post so they could edit/delete the post.